components/form/Select.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ErrorMessage } from './ErrorMessage';
+import { useFormContext } from 'react-hook-form';
+
+export function SelectField({ label, name, ...options }) {
+  const {
+    register,
+    formState: { errors }
+  } = useFormContext();
+
+  const hasErrors = !!errors?.[name];
+
+  const errorClass = hasErrors ? 'focus:border-amcovad-danger' : 'focus:border-amcovad-primary-500';
+
+  const style = {
+    container: `relative my-1 `,
+    default: `${errorClass} relative flex flex-1 w-full mt-1 rounded-md py-2 px-2 bg-transparent text-amcovad-secondary-700 placeholder-amcovad-secondary-700 border border-amcovad-secondary-300 text-base focus:outline-none focus:ring-1 peer border-2 `
+  };
+
+  return (
+    <div className={`${style.container}`}>
+      <label htmlFor={name} className="text-amcovad-secondary-700 text-sm font-Inter ">
```

Comment:
We would want to use the `Label` component here. Please check other form components and ensure that we use the `Label` component for everything.